docs/accelerators.md

```diff
@@ -106,6 +106,31 @@ cd model_server
 make docker_build INSTALL_DRIVER_VERSION=22.10.22597
 ```
 
+## Model Server image with Intel® Data Center GPU Flex Series and Intel® Arc™ GPU support (Ubuntu 20.04)
```

Comment:
This section is not rebased with develop. new public docker image will include support for Arc and Flex. This section is relevant only to try preproduction driver versions